src/platform/windows/display_base.cpp

```diff
@@ -1056,15 +1061,32 @@ namespace platf::dxgi {
 namespace platf {
   std::shared_ptr<display_t>
   display(mem_type_e hwdevice_type, const std::string &display_name, const video::config_t &config) {
+    if (config::video.capture == "wgc") {
```

Comment:
I think there needs to be some changes to the following files to support this.

- `./src/config.cpp` (and config.h?)
- `./src_assets/common/assets/web/config.html`
- `./docs/source/about/advanced_usage.rst`

Reply:
I'm not particularly interested in exposing an option for this in the web UI considering that this feature isn't really ready for general use yet.

I've changed `advanced_usage` to support this.

Reply:
We expose all config options to the UI. If it's not there if will be wiped from the config file when saving from the UI.

You could put `WGP (beta)` though and have the other option as the default.

Reply:
I see! When I implemented this change WGC showed up in the configuration UI, and I only had to modify the html file. This is a brilliant system!